### PR TITLE
multi-tenant setup docs first version

### PR DESCRIPTION
**Pull request by daneryl**

### CODE REVIEW

File: sysadmin-docs/multi-tenant.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  indexName: string; //should be unique
+  uploadedDocuments: string;
+  attachments: string;
+  customUploads: string;
+  temporalFiles: string;
+};
+
+```
+
+uwazi will use this config for requests that provide a Tenant header corresponding with the tenant name, for requests that do not provide one uwazi will use the defaultTenant from app/api/config.ts.
```
Comment (fnocetti) [inline sysadmin-docs/multi-tenant.md]: @daneryl Please, check the capitalization :)

**Review by fnocetti — CHANGES_REQUESTED** [review]
@daneryl It mostly looks good to me! I'll request @nwyu to take a look too!
I left one comment. Also, you need to add this file to the index.md file for the section in order for it to be rendered and visible in the ToC: https://github.com/huridocs/uwazi-documentation/blob/master/sysadmin-docs/index.md

Thank you!

File: sysadmin-docs/multi-tenant.md (hunk 1 above)
Comment (daneryl) [reply sysadmin-docs/multi-tenant.md]: done

**Comment by daneryl** [comment]
@fnocetti i have added the multi-tenant to the toc on sysadmin index.md

**Review by fnocetti — APPROVED** [review]

**Pull request merged**